haddock-api/src/Haddock.hs

```diff
@@ -609,7 +610,8 @@ getPrologue dflags flags =
       h <- openFile filename ReadMode
       hSetEncoding h utf8
       str <- hGetContents h -- semi-closes the handle
-      return . Just $! parseParas dflags Nothing str
+      let stripNs (NsRdrName _ rdrName) = rdrName
```

Comment:
Can we move a utility like `stripNs` next to the definition of `NsRdrName` please